### DISCUSSION TITLE
Tiling Inpaint with Pre-Upscaled Image

### DISCUSSION THREAD

**Discussion by toyssamurai** [Coyote-A/ultimate-upscale-for-automatic1111#124] [Q&A] [answered]
Is there a way to use the tiling inpaint feature with a pre-upscaled image? If I already upscaled an image once, there's little reason to resize it again when I am just not satisfied with the result of the inpaint.

**Comment by Jopezzia** [comment, marked as answer]
Use Scale from image size with 1

**Reply by toyssamurai** [reply]
Thanks!
